package.json

```diff
@@ -35,7 +35,7 @@
   "devDependencies": {
     "@types/jquery": "2.0.41",
     "@types/lodash": "4.14.57",
-    "@types/office-js": "0.0.39",
+    "@types/office-js": "^0.0.42",
```

Comment:
use targeted version to prevent any unwanted issues. Could you update this to `0.0.42` only.